src/reducers/user-reducer.ts

```diff
@@ -51,4 +60,20 @@ export const resetUser = () => {
 	};
 };
 
+/**
+ * Changing tracking/sedning interval in the settings (src/screens/settings-screen.tsx).
+ * Props: newInterval and if the change is the sending or tracking interval
+ * @returns dispatch method to update tracking/sending interval.
+ */
+export const setInterval = (newInterval: number, isTracking: boolean) => {
+	if (isTracking) {
+		return async (dispatch: AppDispatch) => {
+			dispatch(setTrackingInterval(newInterval));
+		};
+	}
+	return async (dispatch: AppDispatch) => {
+		dispatch(setSendingInterval(newInterval));
+	};
+};
+
```

Comment:
From the first glancy this was little bit confusing, because I though that `isTracking` is refering wether route is active or not. Not sure if this is needed, but should we split this into two functions, such as `changeTrackingInterval` and `changeSendingInterval`?